igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/bitbucket/client/model/AbstractBitBucketResponse.groovy

```diff
@@ -17,7 +17,6 @@
 package com.netflix.spinnaker.igor.scm.bitbucket.client.model
 
 abstract class AbstractBitBucketResponse {
-  List<?> values
```

Comment:
just curious, why are you removing this?

Reply:
Because it turns out that not all stash responses have that at the root, including the ones I used for the new APIs.

Reply:
Actually, this was supposed to have been in the stash class only. I think this was IntelliJ trying to be helpful. Will fix — thanks for catching it.

Reply:
On second thought, it turns out this was already defined in the single sub-class, and like stash not all responses will have it at the root, so I'll keep this.